ARM/ARM-Template-Deployment.yml

```diff
@@ -0,0 +1,55 @@
+name: workflow to deploy an ARM Template using CLI commands
+on: push
+
+# CONFIGURATION
+# For help, go to https://github.com/Azure/Actions
+#
+# Set up the following secrets in your repository:
+#   AZURE_CREDENTIALS
+# Change these variables for your configuration:
+#   AZURE_RESOURCE_GROUP, AZURE_LOCATION
+
+env:
+  AZURE_RESOURCE_GROUP: ActionsDemo   # set this to your Azure Resource group's name  
+  AZURE_LOCATION: WestEurope  # set this to your Location Name 
+  
+jobs:
+  build-and-deploy-to-dev:
+    runs-on: ubuntu-latest
+    steps:
+                
+    # Authentication
+    - name: Azure Login
+      uses: azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS  }}
+    
+    # Checkout
+    - name: Checkout
+      uses: actions/checkout@v1
+        
+    # Deployment of template
+    - name: Deploy ARM Template
+      uses: azure/CLI@v1
+      with:
+        azcliversion: latest
+        inlineScript: |
+          az group create --location ${{ env.AZURE_LOCATION }} --name ${{ env.AZURE_RESOURCE_GROUP }}
+          az group deployment create --resource-group ${{ env.AZURE_RESOURCE_GROUP }} --template-file $GITHUB_WORKSPACE/azuredeploy.json
+
+# The inline CLI Script needs to be copied into the yml by the user. Some of the commonly used scripts are listed below:
+#
+# The script for creation of resource group, in case it is not present:
+# az group create --location $Replace_this_with_the_location --name $Replace_this_with_the_resource_group
+#
+# The script for deployment of ARM Template present in the local directory
+# az group deployment create --resource-group $Replace_this_with_the_resource_group --template-file $Replace_this_with_the_template_location
+#
+# The script for deployment of ARM Template from a remote loaction (URI is known)
+# az group deployment create --resource-group $Replace_this_with_the_resource_group --template-uri $Replace_this_with_the_template_URI
+#
+# The script for deployment of ARM Template with parameters file
+# az group deployment create --resource-group $Replace_this_with_the_resource_group --template-uri $Replace_this_with_the_template_URI --parameters $Replace_this_with_parameter_file
+#
```

Comment:
In the intro ( https://github.com/Azure/actions/pull/42/files ) here you mentioned about 3 scenarios. Create, Update and delete. Lets add sample script for delete as well and describe how will update happen.